lib/features/mailbox_dashboard/presentation/controller/mailbox_dashboard_controller.dart

```diff
@@ -126,6 +126,7 @@ class MailboxDashBoardController extends ReloadableController {
   Session? sessionCurrent;
   Map<Role, MailboxId> mapDefaultMailboxIdByRole = {};
   Map<MailboxId, PresentationMailbox> mapMailboxById = {};
+  final emailList = <PresentationEmail>[].obs;
```

Comment:
`emailsInCurrentMailbox` is better, IMO. And please don't forget to reset this list when change the mailbox.